src/data_reader.py

```diff
@@ -49,24 +57,26 @@ def load_data(self, data_type: DataType):
         return data_df
 
     @staticmethod
-    def create_image_payload(data_path: str) -> list[dict]:
```

Comment:
Are we allowing a list of directories as an input now?

Reply:
Oh yeah, I tried with a monthly bank statement, and it failed there. We need to find a sweet spot or probably batch the transactions so that they don't exceed max_tokens.

Reply:
No, it is a list of file paths, so gradio allows users to upload multiple files, and they are temporarily stored (not under a common directory), hence made it a list of file paths.